test/utils/nodes/nodes.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodes
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/k8stopologyawareschedwg/deployer/pkg/clientutil"
```

Comment:
if we want to go with `deployer/pkg/clientutil` (no strong feelings in either direction) we go full swing and just use
https://github.com/fromanirh/deployer/blob/main/pkg/clientutil/nodes/nodes.go -> `GetByRole`

Reply:
or actually `GetWorkers` :)

Reply:
I'm fully aware of this function, but it initializes the Client by itself and not using the one already provided by the ```Fixture``` so I wasn't sure it's such a good idea.

Reply:
There is also this annoying problem:
`Cannot use 'node' (type *"github.com/k8stopologyawareschedwg/deployer/vendor/k8s.io/api/core/v1".Node) as the type *"github.com/openshift-kni/numaresources-operator/vendor/k8s.io/api/core/v1".Node`

Which happens because two packages that import a dependecny with different names

Reply:
fair points. We can keep as it is.